src/config/mod.rs

```diff
@@ -1,24 +1,42 @@
-use std::{env, fs::read_to_string, path::PathBuf};
-
 use crate::{observability, system_contracts};
 
 use crate::config::{
     cache::{CacheConfig, CacheType},
     constants::*,
     show_details::*,
 };
-
+use crate::utils::{format_eth, format_gwei};
+use alloy_signer::Signer;
+use alloy_signer_local::{
+    coins_bip39::{English, Mnemonic},
+    MnemonicBuilder, PrivateKeySigner,
+};
 use cli::Cli;
+use colored::{Colorize, CustomColor};
 use observability::LogLevel;
+use rand::thread_rng;
 use serde::Deserialize;
+use tracing::info;
```

Comment:
Bikeshedding: in our codebase we generally use fully qualified paths for macros, e.g. `tracing::info` or `tokio::spawn`. IIRC anvil/foundry uses imports though, so up to you.

Reply:
Separate PRs for separate things are usually easier to review, but up to you :)